joeynmt/prediction.py

```diff
@@ -460,13 +429,40 @@ def _translate_data(test_data):
                       unk_token=UNK_TOKEN, include_lengths=True)
     src_field.vocab = src_vocab
 
+    params_dict['src_field']=src_field
+
     # parse test args
     batch_size, batch_type, use_cuda, device, n_gpu, level, _, \
         max_output_length, beam_size, beam_alpha, postprocess, \
         bpe_type, sacrebleu, _, _ = parse_test_args(cfg, mode="translate")
```

Comment:
How about returning the dictionary directly by the `parse_test_args` function? This would save us the repeated packing/unpacking.